app/templates/index.html

```diff
@@ -43,9 +43,11 @@
                         <label>Choose your Badge Background</label>
                         <ul style="list-style-type:none">
                             <li>
-                                <input type="radio" name="backsource" id="pngupload">Add PNG file</li>
-								<section id="upimage-input" style="display: none;">
-									<label for="imageFile">Upload a PNG file for badge background</label>
+                                <input type="radio" name="backsource" id="pngupload">Add PNG file
+                                <button type="button" style="background-color:#5BC0DE; font-style:italic; font-weight: bold; border-radius: 85%" data-toggle="tooltip" data-placement="right"
```

Comment:
Please refrain from inline styling. Please visit [here](https://www.thoughtco.com/avoid-inline-styles-for-css-3466846) for more more info

Reply:
I know this and I also don't want to use inline styling but when I am doing it by using a class, it is not getting priority and there is normal button shown, that's why I have used inline .